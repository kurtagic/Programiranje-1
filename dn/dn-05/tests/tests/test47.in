330
8 -571
2 -524
3 -266
2 -557
14 -587
1 -286
2 -553
7 -490
0 -591
0 -200
7 -530
11 -611
11 -729
3 -620
3 -671
3 -302
2 -313
1 -734
7 -149
7 -480
1 -616
1 -387
2 -363
8 -456
8 -675
2 -570
0 -676
8 -521
14 -246
2 -259
8 -436
7 -692
11 -201
11 -715
3 -381
7 -301
2 -621
14 -277
3 -522
3 -199
14 -546
7 -154
2 -559
0 -264
7 -347
7 -363
8 -486
2 -232
3 -689
14 -510
1 -380
11 -303
3 -547
3 -742
1 -563
0 -570
3 -500
14 -363
0 -323
11 -659
7 -544
2 -432
3 -305
3 -554
8 -219
0 -220
7 -750
2 -204
2 -201
14 -622
3 -189
8 -237
1 -372
7 -701
7 -615
11 -455
14 -241
7 -700
0 -366
7 -159
3 -195
8 -504
0 -444
3 -327
3 -436
1 -636
8 -541
2 -729
1 -574
7 -461
1 -516
2 -544
2 -485
8 -139
8 -330
11 -439
0 -623
1 -283
8 -273
14 -578
11 -190
14 -626
8 -334
8 -551
8 -383
8 -507
14 -626
1 -254
14 -705
11 -283
8 -341
0 -282
1 -726
11 -400
14 -195
14 -365
8 -678
7 -412
0 -353
8 -727
8 -427
8 -489
0 -153
11 -468
3 -179
7 -217
8 -533
3 -427
1 -204
8 -306
3 -283
11 -283
11 -607
3 -274
3 -528
3 -687
3 -380
7 -228
3 -227
3 -427
3 -548
2 -434
2 -413
2 -210
1 -451
1 -135
2 -564
8 -526
2 -355
0 -186
2 -711
0 -143
8 -520
14 -517
3 -253
14 -538
8 -445
7 -414
11 -334
2 -470
8 -571
8 -215
11 -351
14 -247
8 -747
1 -163
8 -382
0 -400
3 -204
8 -184
1 -169
1 -252
2 -615
3 -161
7 -580
7 -463
14 -665
0 -572
2 -217
7 -635
1 -193
2 -655
0 -578
2 -293
2 -718
3 -693
3 -538
8 -279
2 -695
0 -713
0 -556
11 -380
7 -449
14 -586
0 -240
3 -740
7 -213
2 -742
3 -487
0 -383
14 -377
3 -240
7 -724
8 -516
14 -531
7 -309
3 -633
14 -711
0 -326
2 -249
8 -196
3 -515
0 -145
8 -382
0 -635
8 -145
1 -701
3 -685
14 -526
7 -385
3 -564
2 -230
3 -497
2 -449
0 -659
8 -561
7 -234
7 -241
8 -153
11 -592
14 -722
0 -243
1 -266
1 -610
1 -228
1 -557
7 -302
14 -159
2 -174
14 -333
0 -654
3 -521
7 -141
0 -565
3 -262
8 -136
14 -386
0 -403
7 -566
14 -600
1 -273
0 -620
0 -336
3 -445
7 -597
7 -181
11 -218
3 -323
8 -164
1 -550
14 -742
0 -640
0 -215
3 -742
1 -361
2 -379
2 -158
11 -712
2 -576
7 -689
0 -628
7 -277
7 -488
0 -346
7 -351
3 -553
7 -452
0 -596
0 -539
0 -164
3 -148
7 -502
11 -467
2 -239
7 -378
8 -703
0 -411
14 -698
2 -209
11 -622
0 -339
1 -499
11 -297
0 -363
8 -272
8 -538
2 -520
14 -362
0 -437
7 -698
14 -642
8 -285
11 -452
11 -368
0 -532
11 -622
1 -685
7 -523
1 -629
1 -502
11 -289
2 -488
11 -323
14 -477
2 -218
11 -621
1 -453
0 -344
2 -293
11 -207
2 -360
14 -633
1 -394
11 -642
7 -333
0 -558
1 -176
7 -235
1 -416
3 -672
