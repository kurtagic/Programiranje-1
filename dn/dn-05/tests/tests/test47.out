-750: 3
-749: 1
-747: 1
-746: 1
-745: 2
-742: 7
-741: 8
-740: 7
-739: 3
-738: 2
-734: 4
-729: 5
-728: 5
-727: 5
-726: 5
-725: 6
-724: 3
-723: 1
-722: 2
-721: 1
-720: 1
-718: 2
-717: 2
-715: 1
-714: 3
-713: 3
-712: 6
-711: 8
-710: 4
-709: 4
-705: 2
-704: 1
-703: 8
-702: 8
-701: 9
-700: 4
-699: 2
-698: 5
-697: 3
-696: 1
-695: 2
-694: 2
-693: 4
-692: 5
-691: 4
-689: 5
-688: 3
-687: 2
-686: 3
-685: 7
-684: 4
-683: 3
-678: 4
-677: 4
-676: 5
-675: 3
-674: 3
-673: 3
-672: 3
-671: 4
-670: 3
-669: 1
-665: 2
-664: 1
-663: 1
-659: 4
-658: 4
-657: 4
-656: 4
-655: 2
-654: 3
-653: 3
-652: 3
-651: 3
-642: 4
-641: 6
-640: 2
-639: 1
-638: 1
-637: 1
-636: 4
-635: 4
-634: 4
-633: 6
-632: 5
-631: 5
-629: 4
-628: 5
-627: 5
-626: 5
-625: 5
-624: 3
-623: 6
-622: 10
-621: 13
-620: 15
-619: 7
-618: 7
-617: 7
-616: 4
-615: 5
-614: 5
-611: 1
-610: 7
-607: 1
-606: 3
-600: 2
-599: 1
-598: 1
-597: 3
-596: 2
-595: 2
-594: 2
-593: 2
-592: 2
-591: 4
-590: 1
-589: 1
-588: 1
-587: 2
-586: 3
-585: 2
-584: 2
-580: 3
-579: 1
-578: 3
-577: 3
-576: 5
-575: 5
-574: 4
-572: 8
-571: 8
-570: 8
-569: 8
-568: 5
-567: 5
-566: 12
-565: 10
-564: 9
-563: 9
-562: 9
-561: 3
-560: 3
-559: 4
-558: 7
-557: 7
-556: 7
-555: 7
-554: 5
-553: 6
-552: 7
-551: 6
-550: 5
-549: 2
-548: 5
-547: 6
-546: 5
-545: 3
-544: 6
-543: 6
-541: 1
-540: 1
-539: 5
-538: 6
-537: 6
-536: 7
-533: 1
-532: 6
-531: 6
-530: 6
-529: 6
-528: 1
-527: 2
-526: 6
-525: 5
-524: 5
-523: 5
-522: 3
-521: 4
-520: 7
-519: 7
-518: 4
-517: 6
-516: 6
-515: 8
-514: 9
-513: 8
-510: 2
-509: 1
-508: 1
-507: 1
-506: 1
-505: 2
-504: 1
-503: 1
-502: 9
-501: 3
-500: 1
-499: 6
-498: 1
-497: 1
-496: 2
-495: 1
-490: 3
-489: 2
-488: 10
-487: 10
-486: 6
-485: 5
-484: 4
-480: 3
-479: 1
-477: 2
-476: 1
-475: 1
-470: 2
-469: 2
-468: 1
-467: 4
-466: 6
-463: 3
-462: 1
-461: 3
-460: 1
-456: 1
-455: 3
-454: 5
-453: 4
-452: 8
-451: 10
-449: 5
-448: 5
-445: 4
-444: 5
-443: 4
-442: 1
-441: 1
-439: 1
-438: 3
-437: 6
-436: 6
-435: 6
-434: 6
-433: 5
-432: 2
-431: 2
-427: 5
-426: 6
-425: 5
-416: 4
-414: 8
-413: 6
-412: 5
-411: 2
-410: 2
-409: 2
-408: 2
-403: 5
-402: 5
-401: 5
-400: 5
-399: 4
-398: 4
-397: 4
-394: 4
-387: 4
-386: 5
-385: 4
-384: 4
-383: 11
-382: 12
-381: 12
-380: 13
-379: 14
-378: 17
-377: 15
-376: 1
-375: 1
-372: 4
-368: 1
-367: 3
-366: 8
-365: 10
-364: 9
-363: 11
-362: 13
-361: 12
-360: 14
-359: 14
-355: 2
-354: 2
-353: 1
-352: 1
-351: 7
-350: 5
-347: 3
-346: 2
-345: 2
-344: 3
-343: 3
-342: 3
-341: 3
-340: 1
-339: 8
-338: 8
-337: 8
-336: 8
-335: 7
-334: 7
-333: 10
-332: 8
-331: 5
-330: 1
-329: 1
-328: 2
-327: 1
-326: 3
-325: 3
-324: 3
-323: 6
-322: 8
-321: 4
-320: 1
-313: 2
-312: 2
-309: 3
-308: 1
-306: 5
-305: 5
-304: 6
-303: 3
-302: 8
-301: 6
-300: 3
-297: 1
-296: 3
-293: 4
-292: 4
-289: 1
-288: 3
-286: 4
-285: 12
-284: 12
-283: 13
-282: 13
-281: 9
-280: 8
-279: 9
-278: 9
-277: 13
-276: 11
-275: 1
-274: 2
-273: 7
-272: 3
-271: 3
-270: 4
-266: 5
-265: 2
-264: 2
-263: 2
-262: 3
-261: 4
-260: 3
-259: 2
-258: 2
-254: 4
-253: 1
-252: 6
-251: 1
-249: 2
-248: 2
-247: 4
-246: 3
-245: 3
-244: 1
-243: 7
-242: 7
-241: 7
-240: 7
-239: 6
-238: 6
-237: 2
-236: 1
-235: 6
-234: 4
-233: 1
-232: 2
-231: 2
-230: 2
-229: 2
-228: 7
-227: 2
-226: 3
-225: 2
-220: 3
-219: 3
-218: 13
-217: 13
-216: 8
-215: 6
-214: 6
-213: 6
-212: 6
-210: 2
-209: 4
-208: 4
-207: 1
-206: 3
-204: 7
-203: 8
-202: 7
-201: 6
-200: 6
-199: 2
-198: 3
-197: 2
-196: 5
-195: 5
-194: 6
-193: 7
-190: 2
-189: 4
-188: 2
-187: 1
-186: 1
-185: 1
-184: 2
-183: 2
-182: 3
-181: 3
-180: 1
-179: 1
-178: 2
-177: 1
-176: 4
-174: 2
-173: 2
-169: 4
-164: 7
-163: 7
-162: 7
-161: 7
-160: 5
-159: 6
-158: 7
-157: 7
-154: 3
-153: 3
-152: 3
-151: 4
-150: 2
-149: 3
-148: 2
-147: 3
-146: 2
-145: 3
-144: 3
-143: 4
-142: 2
-141: 4
-140: 2
-139: 1
-138: 1
-137: 2
-136: 5
-135: 5
-134: 6
